include/llvm/Config/config.h

```diff
@@ -18,7 +18,7 @@
 #define ENABLE_CRASH_OVERRIDES 1
 
 /* Define to 1 if you have the `backtrace' function. */
-#if !defined(__ANDROID__) && !defined(__OpenBSD__)
+#if !defined(__ANDROID__) && !defined(__OpenBSD__) && defined(__GLIBC_PREREQ)
```

Comment:
Maybe this should be `#if __has_include(<execinfo.h>)`? That's where `backtrace` is declared, and would avoid "knowing" that it doesn't exist on Android or OpenBSD.